CIP-0143/README.md

```diff
@@ -0,0 +1,154 @@
+1. CIP 68 Smart Tokens
+2. Transfer Scripts (ledger changes required)
+
+The CIP 68 approach allows the tokens to be used anywhere but if the contract does not obey the logic then the token can be invalidated (ie revoked). So if you put it into a liquidity pool and the batcher does not obey the token logic then your tokens can be revoked and it wouldn't be your fault.
+The transfer scripts proposal is to introduce a new Plutus script type that would need to be invoked in any transaction that spends a smart token (identified by the policy id). The issue with this approach is that it introduces a huge potential for vulnerabilities and exploits to the existing ledger and these vulnerabilities are responsible for the vast majority of exploits and centralization risks in ecosystems with fully programmable tokens (ie Ethereum). Regardless, this means that these tokens could not be used on existing dApps, since they would require a new Plutus version with new features, and the ledger does not allow contracts that use new features to co-exist in transactions with contract from previous versions where those features did not exist. 
+
+Furthermore, all of the aforementioned proposals would require custom indexers and infrastructure to locate a user's (or smart contract's) programmable tokens. You could not simply query an address, instead you would need to query UTxOs from the contracts and check their datum / value (depending on the CIP) to determine the owner.
+
+The above factors motivated the design of this framework. Some of the core unique properties of this framework include: 
+1. Each users gets their own smart tokens address (that can be derived from staking keys, pub key hashes, and script hashes)
+2. Interoperability with existing dApps
+3. Introduces no new risk / vulnerabilities into existing protocols
+4. Doesn't require changes to the ledger
+5. Smart tokens cannot be revoked by dApps that fail to follow the standard (unlike the CIP 68 case in which they can)
+6. Very low effort (relative to the existing proposals) to implement and get it to production / mainnet adoption
+7. Completely permissionless and natively interoperable with other smart tokens. IE anyone can mint their own smart tokens with their own custom logic and the correctness of their behavior will be enforced
+
+
+## Path to Active
+
+### Acceptance Criteria
+Issuance of at-least one smart token via the proposed framework on the following networks:
+1) Preview testnet
+2) Mainnet
+
+End-to-end tests of programmable token logic including arrestability, transfer fees, and blacklisting. 
+
+Finally, a widely adopted wallet that can read and display programmable token balances to users and allow the user to conduct transfers of such tokens. 
+
+### Implementation Plan
+Implement the contracts detailed in the specification, and implement the offchain code required to query programmable token balances and construct transactions to transfer such tokens. 
+
```

Comment:
```suggestion
### Acceptance Criteria
- [ ] Issuance of at-least one smart token via the proposed framework on the following networks:
  - [ ] 1. Preview testnet
  - [ ] 2. Mainnet
- [ ] End-to-end tests of programmable token logic including arrestability, transfer fees, and blacklisting. 
- [ ] Finally, a widely adopted wallet that can read and display programmable token balances to users and allow the user to conduct transfers of such tokens. 

### Implementation Plan
- [ ] Implement the contracts detailed in the specification.
- [ ] Implement the offchain code required to query programmable token balances and construct transactions to transfer such tokens. 
```
also fed back from yesterday's CIP meeting: adding tick-boxes for checklists which are bound to become more detailed & also include a possible dependency on a related `Plutus` category CIP.

Reply:
@colll78 I'm committing this so I can confirm that the syntax is formatted correctly.